setup-ovs.sh

```diff
@@ -41,14 +41,15 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
   ovs-vsctl --if-exists del-br brcnv
```

Comment:
can you amend the commit description to comply with the github style?
Mainly, subject line should be less than 72 characters, and the message body should be wrapped after 72 as well.
Suggestion:
```
fix race when configuring 802-3-ethernet.cloned-mac-address argument

As per BZ[1] the MAC should be set for the interface, not the bridge, 
to prevent a race condition.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1899745#c2
```